src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -10711,10 +10732,13 @@ definitions:
       attachment:
         type: boolean
         description: Flag to control the download method, **true** for attachment URL, **false** for embedded URL
-      expiresAt:
+      expiryTime:
```

Comment:
Please revert the field to `expiresAt`. The name is based on our RestAPI guidelines. Ideally we should fix the attribute name in the `DirectAccessUrl` class but since it is part of our Java Public APIs we can't just rename it...

Reply:
Done!